crates/wasmtime/src/compile/call_graph.rs

```diff
@@ -0,0 +1,103 @@
+//! Construction of the call-graph.
```

Comment:
We should clarify the scope and soundness/completeness of the call graph here -- in particular, with respect to imports and indirect calls (one could pick any of several set-points to get e.g. an overapproximation that's sound, or an underapproximation that gives partial but exact answers, and I think inlining wants the latter)